WolvenKit.Modkit/RED4/Unbundle.cs

```diff
@@ -59,7 +64,16 @@ public void ExtractAll(Archive ar, DirectoryInfo outDir, string pattern = "", st
             }
 
             using var fs = new FileStream(ar.ArchiveAbsolutePath, FileMode.Open, FileAccess.ReadWrite, FileShare.ReadWrite);
```

Comment:
Should this be moved into the catch clause, or is that not a problem?

Reply:
Yes, that can be moved into the catch clause.  Let me go ahead and move it.